dashboard/src/main/webapp/src/main.js

```diff
@@ -21,8 +21,10 @@ import ElementPlus from 'element-plus'
 import * as ElementPlusIconsVue from '@element-plus/icons-vue'
 import 'element-plus/dist/index.css'
 import router from "@/router";
+import dayjs from "dayjs"
+
 const app = createApp(App)
 Object.keys(ElementPlusIconsVue).forEach(key => {
     app.component(key, ElementPlusIconsVue[key])
 })
-app.use(router).use(ElementPlus).mount('#app')
+app.use(router).use(ElementPlus).use(dayjs).mount('#app')
```

Comment:
Is this modification necessary? Does every dependency added need to be modified like this in the future?

Reply:
> Is this modification necessary? Does every dependency added need to be modified like this in the future?

The public method has been extracted, and this form is not used.